### ISSUE TITLE
Collections: Create an admin show view

### ISSUE THREAD

**Issue by elrayle** [samvera/hyrax#1427]
### Descriptive summary

Create code that implements the admin show view for a single collection.

### Rationale

See model issue for rationale.

### Implementation

**Create File:** 
* [app/views/hyrax/admin/collections/show.html.erb](https://github.com/samvera/hyrax/blob/master/app/views/admin/hyrax/collections/show.html.erb) - NEW

Probably copy files from [app/views/hyra/collections](https://github.com/samvera/hyrax/blob/master/app/views/hyrax/collections) as a starting point.  May want to complete the landing page issue first as some of those changes will be made in the `app/views/hyrax/collections` directory and will apply to the admin version too.

Will need to...
- [x] create show files under `app/views/hyrax/dashboard/collections`
- [ ] UI
  - [x] show thumbnail
  - [x] include link to public landing page
  - [x] include descriptive metadata according to presenter
  - [x] include list of member items
- [x] update routes to add in a route to the admin show page
- [x] update controller to process actions for the admin show page
- [x] create a presenter controlling the display of metadata, etc. on the admin show page
- [x] update Your/All Collections links to collections to go to this admin show page for the collection

The primary checklist for the UI is the mockup below.

#### SPECIAL NOTE on Admin Sets:

For the initial implementaiton, the Admin Set underlying code will not change.  But to a user, the Admin Set will appear to be a collection.

To make the illusion happen...
* At least initially, the show page for admin sets will NOT change.  We can revisit if time allows.

### Mockup

![admin-collection-show](https://user-images.githubusercontent.com/101482/28979760-dc6bde68-78ff-11e7-8a35-3f859bebdd9f.png)

### Related UI work

#1427 - PR #1562 - Add content to dashboard collections show page
#1552 - PR #1632 - Add new work to collection from collection show page
#1586 - PR #1629 - Show page should only provide Remove action for works and sub-collections
#1678 - Collections: Make admin show page match mockup

### Related work

| Collections feature | Issue |
| ------------------------ | ----- |
| Routes                   | #1380 |
| Controller               | # |
| Model                    | #1383 |
| View - Index            | #1417 |
| View - Show - Admin      | (this issue) |
| View - Show - Landing    | #1428 |
| View - New/Edit          | #1374 |
| Partial -  Description   | #1375 |
| Partial -  Branding      | #1376 |
| Partial -  Relationships | #1377 |
| Partial -  Sharing       | #1378 |
| Partial -  Discovery     | #1379 |

**Comment by elrayle** [comment]
Based on the recommendation of @mjgiarlo, the location of the show file will move out of the admin directory and into the dashboard directory.

/app/views/hyrax/dashboard/show.html.erb.

**Comment by laritakr** [comment]
blocked by issue #1380

**Comment by elrayle** [comment]
PR #1499 which was rebased into collections-sprint branch freed up the block.

**Comment by elrayle** [comment]
Also related to this work...  #1545 Collections: index page - add processing that allows for a work to be created directly in this collection

NOTE: It would be good to coordinate with @hortongn on the adding of the `Deposit new works through this collection` button.  Having that button is a prerequisite for #1545

**Comment by elrayle** [comment]
It looks like all work as specified in this issue is complete.  Need to review the UI in the test app against the mockup to see if any tickets need to be created to tweak the UI.

NOTE: Issue #1545 is still open at this time.

**Comment by elrayle** [comment]
Need issues created for remaining work identified as part of the partial PR #1562 

This is a partial PR to rearrange the layout of the dashboard/show page and add some new content.

* It does not include CSS changes to make the layout pretty.
* It does not accommodate the inclusion of new data in the list of works shown.
* It includes a stubbed collection_type_badge method, but does not yet retrieve the actual collection type or format the badge layout.

**Comment by elrayle** [comment]
Remaining Work:

* Leaving this issue open and putting in ready column for a UI person to make the show page match the mockup.

**Comment by elrayle** [comment]
All requirements of this issue are either met or represented in another issue.  Closing.
